micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/fluent/common/response/executor/RestExecutor.groovy

```diff
@@ -118,7 +118,7 @@ final class RestExecutor<T> {
 
                     @Override
                     protected ResponseEntity<T> getFallback() {
-                        return hystrixFallback.call()
+                        return wrapWithResponseEntity(hystrixFallback.call())
```

Comment:
Maybe it is worth to add comment next to `@TypeChecked` to not replace it with `@CompileStatic` due to dynamic dispatch?